src/ontology/uberon.Makefile

```diff
@@ -1276,7 +1277,8 @@ $(MAPPINGDIR)/biomappings.sssom.tsv: $(TMPDIR)/biomappings.sssom.tsv \
 	sssom-cli --input $< --prefix 'UBERON=http://purl.obolibrary.org/obo/UBERON_' \
 		  --rule '!(subject==UBERON:* || object==UBERON:*) -> stop()' \
 		  --rule 'object==UBERON:* -> invert()' \
-		  --include-all | \
+		  --include-all \
+		  --update-from-ontology=$(SRC):subject,label,existence | \
```

Comment:
Nothing here looks particularly wrong or strange, but maybe a super shart explainer of this new option. I read https://incenp.org/dvlpt/sssom-java/sssom-cli.html, very nice;

> If the subject or the object of a mapping does not exist in the ontology or is deprecated, the mapping is removed from the set.


In order for "existence" to be fulfilled, what are the exact conditions?

1. An OWL declaration exists
2. The ID is used in an axiom
3. The ID is used as the object of an xref assertion

Thanks!

Reply:
> In order for "existence" to be fulfilled, what are the exact conditions?
>
> 1. An OWL declaration exists
> 2. The ID is used in an axiom

The IRI of the entity (in this case, the mapping’s `subject_id`) must be in the ontology’s signature. This means that the IRI must either:

1. be the subject of an OWL declaration axiom, or
2. be used in at least one axiom **that is enough to allow inferring the type of entity** (e.g. a _SubClassOf_ axiom to infer that the referred entity is a class).

(In fact condition (1) is merely a special case of condition (2): a declaration axiom is precisely “an axiom that is enough to allow inferring the type of the entity”.)

Reply:
So for example, if the only axiom the ontology contains about entity `EX:0001` is a label annotation like this:

```
AnnotationAssertion(rdfs:label EX:0001 "my class")
```

then we don’t know what `EX:0001` is (class, property, individual?) and so it is not considered part of the signature (at least this is how the OWLAPI sees it). Therefore, from SSSOM-CLI’s point of view, that entity would be considered _not_ to exist, and under the `existence option a mapping with EX:0001 would be removed.

Reply:
Just as an irrelevant aside:

> (In fact condition (1) is merely a special case of condition (2): a declaration axiom is precisely “an axiom that is enough to allow inferring the type of the entity”.)

didn't used to be true. Formally, yes, but not practically. Declarations were injected and write-time if they were missing.. `x.signature()` methods behave correctly though!